src/recovery.rs

```diff
@@ -380,21 +387,28 @@ impl Recovery {
         self.rate_sample.delivery_rate
     }
 
-    fn update_rtt(&mut self, latest_rtt: Duration, ack_delay: Duration) {
+    fn update_rtt(
+        &mut self, latest_rtt: Duration, ack_delay: Duration, now: Instant,
+    ) {
+        let update_time = now;
```

Comment:
This variable is not really needed at all, you can pass `now` directly into `reset()` and `running_min()` AFAICT.

Reply:
done!